skymap_scanner/recos/millipede_wilks.py

```diff
@@ -47,31 +45,28 @@ class MillipedeWilks(RecoInterface):
     pulsesName = cfg.INPUT_PULSES_NAME + "IC"
     pulsesName_cleaned = pulsesName+'LatePulseCleaned'
 
-    # Load Data ########################################################
+    def __init__(self):
+        pass
 
-    # At HESE energies, deposited light is dominated by the stochastic losses
-    # (muon part emits so little light in comparison)
-    # This is why we can use cascade tables
-    datastager = DataStager(
-        local_paths=cfg.LOCAL_DATA_SOURCES,
-        local_subdir=cfg.LOCAL_SPLINE_SUBDIR,
-        remote_path=f"{cfg.REMOTE_DATA_SOURCE}/{cfg.REMOTE_SPLINE_SUBDIR}",
-    )
+    def setup_reco(self):
```

Comment:
Can you add a doc block indicating that this method is used to load expensive resources?

Reply:
Will do in `RecoInterface`.